### PR TITLE
Improve SEO

### PR DESCRIPTION
**Pull request by Pandapip1**
## Summary

The EIPs repository is already about as close to having a perfect score as one can get, but this will actually make the score perfect.

## Old
image

## New

image

Thanks for the confetti, lighthouse! 🎉

### CODE REVIEW

**Comment by Pandapip1** [comment]
I'm also considering adding another PR that enables PWA. Offline EIP viewing and backups go brr!

**Comment by xinbenlv** [comment]
This is great improvement. Thank you for the effort!

File: assets/css/style.scss [hunk 1]
```diff
@@ -30,6 +30,22 @@ main.page-content {
   }
 }
 
+.footer-col-wrapper {
+  color: #111;
+}
+
+a, a:link, a:visited {
```
Comment (SamWilsn) [inline assets/css/style.scss:37]: Any particular reason you're making visited links the same color? I actually like them being different.

Comment (Pandapip1) [reply assets/css/style.scss:37]: Not really. Do you have a good color in mind?

Comment (SamWilsn) [reply assets/css/style.scss:37]: `#8E6680`?

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**